.github/workflows/check-catalyst.yaml

```
@@ -31,7 +31,6 @@ jobs:
         run: |
           COMPILER_LAUNCHER="" \
           RT_BUILD_DIR="$(pwd)/runtime-build" \
-          ENABLE_LIGHTNING_KOKKOS=ON \
           CMAKE_ARGS="-DENABLE_SANITIZER=ON" \
```

What about turning it OFF instead of removing it?
```suggestion
          ENABLE_LIGHTNING_KOKKOS=OFF \
          CMAKE_ARGS="-DENABLE_SANITIZER=ON" \
```

---

sure